suites/tentacle/nvmeof/tier-3_nvmeof_scale_ns.yaml

```diff
@@ -0,0 +1,214 @@
+# Test module to scale Single GW components - scale factor being 1 subsystem and 256 namespaces
```

Comment:
Please remove this suite as well as now we combined scale test suites with GW groups